components/expenses/Expense.js

```diff
@@ -171,6 +171,14 @@ class Expense extends React.Component {
           (Date.now() - new Date(expense.updatedAt).getTime() < 60 * 1000 * 15 || // admin of collective can reject the expense for up to 10mn after approving it
             LoggedInUser.canEditCollective(collective.host))));
 
+    let precision = 0;
+
+    // Check if the remainder is not 0, this check is important in order to know the amount that needs precision
+    // i.e display $10.21 as $10.21 instead of $10
+    if ((expense.amount / 100) % 1 !== 0) {
+      precision = 2;
+    }
+
```

Comment:
@znarf what do you think about this? Is that something we want to have generic in `AmountCurrency` or should we always show `$10.00` when precision is 2?

Reply:
@Betree I approved this change but I'm good to rethink this. I think we should get a team wide consensus on rounding before moving forward. Make sure we get a decision and don't discuss that again in the future.

Reply:
Opened an issue to follow up on this discussion. https://github.com/opencollective/opencollective/issues/2579